plugin/pkg/scheduler/algorithm/types.go

```diff
@@ -54,10 +54,15 @@ type PriorityConfig struct {
 	Weight   int
 }
 
-func EmptyMetadataProducer(pod *api.Pod, nodeNameToInfo map[string]*schedulercache.NodeInfo) interface{} {
-	return nil
+// emptyMetadataProducer returns a no-op MetadataProducer.
```

Comment:
i dug into it , i think its simpler if i just revert this change.   found a case yest where some casting was required that looked ugly to me, but its not going to effect the tests from passing